src/Hermes/Models/DatabaseNotification.php

```diff
@@ -58,6 +58,10 @@ public function content(): ?string
 
     public function link(): ?string
     {
+        if (data_get($this->data, 'action.use_action_url', false)) {
```

Comment:
```suggestion
        if (data_get($this->data, 'action.force', false)) {
```

maybe change this to `force`? also avoids having `_` in the name. camelCase is what we would normally use for properties, unless it's directly related to the database where columns are using snake_case. With these notifications it's a bit of both, so if `force` is not a good word it should be changed to `useActionUrl`

Reply:
🤔 on second thought it could stay snake_case as I see there's not too much consistency with those properties at this time; will be good to go over usage in general and tidy it up then

Reply:
@ItsANameToo I like `Force`. It describes it much better 👍🏻